src/app/ccc/sitemap.ts

```diff
@@ -14,7 +14,7 @@ export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
   /* Families */
 
   const familiesData = await fetch(`https://api.climatepolicyradar.org/families/?${urlSearchParams.toString()}`).then((resp) => resp.json());
-  const familiesSiteMap = familiesData.data.map((family) => {
+  const familiesSiteMap = familiesData.data.map((family: any) => {
```

Comment:
Critique: I'll abide this for now, but I would like the final strict mode pass to be refactoring out as many `any` uses as possible - they turn off the TypeScript typing system wherever they're passed around, so they're infectious.